diff --git a/node_modules/amazon-cognito-identity-js/ios/RNAWSCognito.h b/node_modules/amazon-cognito-identity-js/ios/RNAWSCognito.h
index a28e3c1..ab6fdce 100644
--- a/node_modules/amazon-cognito-identity-js/ios/RNAWSCognito.h
+++ b/node_modules/amazon-cognito-identity-js/ios/RNAWSCognito.h
@@ -1,20 +1,6 @@
-#if __has_include("RCTBridgeModule.h")
-#import "RCTBridgeModule.h"
-#else
 #import <React/RCTBridgeModule.h>
-#endif
-
-#if __has_include("RCTLog.h")
-#import "RCTLog.h"
-#else
 #import <React/RCTLog.h>
-#endif
-
-#if __has_include("RCTUtils.h")
-#import "RCTUtils.h"
-#else
 #import <React/RCTUtils.h>
-#endif
 
 #import "JKBigInteger.h"
 
